tests/unit/repo_setup/test_main.py

```diff
@@ -611,21 +530,21 @@ def test_inject_mirrors_centos(self):
         start_repo = '''
 [delorean]
 name=delorean
-baseurl=https://trunk.rdoproject.org/centos7/some-repo-hash
+baseurl=https://trunk.rdoproject.org/centos9/some-repo-hash
 enabled=1
 [centos]
 name=centos
-baseurl=http://mirror.centos.org/centos/7/virt/$basearch/kvm-common
+baseurl=http://mirror.centos.org/centos/9/virt/$basearch/kvm-common
```

Comment:
Doesnt resolve .... https://mirror.stream.centos.org/9-stream/

Reply:
Test data no longer needed. Dropped from the code.